src/olympia/translations/query.py

```diff
@@ -102,13 +108,9 @@ class SQLCompiler(compiler.SQLCompiler):
     def get_from_clause(self):
         # Temporarily remove translation tables from query.tables so Django
         # doesn't create joins against them.
```

Comment:
That comment needs to be updated well - just mentioning `query.alias_map` instead of `query.tables` should be enough.